rest-api-spec/src/main/resources/rest-api-spec/test/range/10_basic.yml

```diff
@@ -391,3 +391,59 @@ setup:
         body: { "size" : 0, "query" : { "range" : { "date_range" : { "gte": "2017-09-03", "lte" : "2017-09-04", "relation": "within" } } } }
 
   - match: { hits.total: 0 }
+
+---
+"Date range rounding":
+  - skip:
+      version: " - 7.99.99"
+      reason:  "This part tests rounding behaviour changed in 8.0.0"
+
+  - do:
+      index:
+        index: test
+        id: 1
+        body: { "date_range" : { "gte": "2019-12-14T12:00:00.000Z", "lte": "2019-12-14T13:00:00.000Z" } }
+
+  - do:
+      index:
+        index: test
+        id: 2
+        body: { "date_range" : { "gte": "2019-12-15T12:00:00.000Z", "lte": "2019-12-15T13:00:00.000Z" } }
+
+  - do:
+      index:
+        index: test
+        id: 3
+        body: { "date_range" : { "gte": "2019-12-16T12:00:00.000Z", "lte": "2019-12-16T13:00:00.000Z" } }
+
+
+  - do:
+      indices.refresh: {}
+
+  - do:
+      search:
+        rest_total_hits_as_int: true
```

Comment:
this parameter is supposed to only exist for the 6.x -> 7.x transition, let's use `track_total_hits: true` instead?

Reply:
Just saw that “rest_total_hits_as_int” is still used all over the place in the yml tests, also in other tests in same test suite. I agree it would be good to get rid of it when its supposed to go away, but I’d like to keep it for consistency with the rest atm. Maybe we should open an issue to clean up all of those in a separate PR?